posts/7-common-errors-to-check-when-debugging-airflow-dag.md

```diff
@@ -1,223 +1,203 @@
 
-To properly trigger your DAG to run, make sure to insert a fixed time in the past (e.g. datetime(2019,1,1)) and set [catchup=False](https://github.com/apache/airflow/blob/v1-9-stable/airflow/models.py#L2865) (unless you're looking to run a backfill).
+To properly trigger your DAG to run, make sure to insert a fixed time in the past and set [catchup=False](https://github.com/apache/airflow/blob/v1-9-stable/airflow/models.py#L2865) if you don't want to perform a backfill.
 
-**Note:** You can manually trigger a DAG run via Airflow's UI directly on your dashboard (it looks like a "Play" button). A manual trigger executes immediately and will _not_ interrupt regular scheduling, though it will be limited by any concurrency configurations you have at the DAG, deployment level or task level. When you look at corresponding logs, the `run_id` will show `manual__` instead of `scheduled__`.
+> **Note:** You can manually trigger a DAG run via Airflow's UI directly on your dashboard (it looks like a "Play" button). A manual trigger executes immediately and will _not_ interrupt regular scheduling, though it will be limited by any concurrency configurations you have at the DAG, deployment level or task level. When you look at corresponding logs, the `run_id` will show `manual__` instead of `scheduled__`.
 
-## 3. You're seeing a 503 Error on your deployment
+## 3. You're seeing a 503 Error on your Deployment.
 
-If you hop into your Airflow deployment only to realize that your instance is entirely inaccessible via your web browser, it very likely has something to do with your Webserver. 
+If you hop into your Airflow Deployment only to realize that your instance is entirely inaccessible via your web browser, it very likely has something to do with your Webserver. 
 
 If you've already refreshed the page once or twice and continue to see a 503 error, read below for some Webserver related guidelines.
 
-#### Your Webserver might be crashing
+### Your Webserver might be crashing.
 
-A 503 error is generally indicative of an issue with your deployment's Webserver, the core Airflow component responsible for rendering task state and task execution logs in the Airflow interface. If it's underpowered or otherwise experiencing an issue for any reason, you can generally expect it to affect UI loading time or web browser accessibility.
+A 503 error is generally indicative of an issue with your Deployment's Webserver, the core Airflow component responsible for rendering task state and task execution logs in the Airflow UI. If it's underpowered or otherwise experiencing an issue for any reason, you can generally expect it to affect UI loading time or web browser accessibility.
```

Comment:
```suggestion
A 503 error might indicate an issue with your Deployment's Webserver, which is the core Airflow component responsible for rendering task state and task execution logs in the Airflow UI. If it's underpowered or otherwise experiencing an issue, you can expect it to affect UI loading time or web browser accessibility.
```